doc/dftb+/manual/manual.bib

```diff
@@ -412,6 +412,30 @@ @Article{grimme-jcp-32-1456-1465
  pages = {1456-1465}
 }
 
+@article{caldweyher-jcp-147-034112,
+author = {Eike Caldeweyher and Christoph Bannwarth and Stefan Grimme},
+title = {Extension of the D3 dispersion coefficient model},
+journal = {J. Chem. Phys.},
```

Comment:
The is a defined string in titles.bib for this journal, so
journal = jcp,
works (and handles the spacing correctly on 'Chem. ').